### DISCUSSION TITLE
A tutorial on how to create block variations

### DISCUSSION THREAD

**Discussion by ndiego** [WordPress/developer-blog-content#154] [Topic Ideas]
This needs a catchy title, but the article would cover:

- What are block variations
- How to create block variations
- Why you would want to use block variations for your own projects
- 3-4 real-world examples of block variations

Currently, there isn't a dedicated Dev Blog article on block variations. This article would serve as a foundational piece of content. Future articles could explore more sophisticated implementations of variations while referring back to this article for the basics.

**Comment by marybaum** [comment]
How about 

Everything you ever wanted to know about block variations—but were afraid to ask

?

**Comment by abhansnuk** [comment]
This is a needed topic on the blog.

**Comment by bph** [comment]
Conversation locked, please continue discussion on the issue: 

- #155

**Comment by ndiego** [comment]
Published article: https://developer.wordpress.org/news/2023/08/an-introduction-to-block-variations/
